### PR TITLE
i18n: use placeholders more for html terms

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: lighthouse-core/audits/uses-rel-preconnect.js [hunk 1]
```diff
@@ -36,17 +36,17 @@ const UIStrings = {
...
    * @description A warning message that is shown when the user tried to follow the advice of the audit, but it's not working as expected. Forgetting to set the `crossorigin` HTML attribute, or setting it to an incorrect value, on the link is a common mistake when adding preconnect links.
    * @example {https://example.com} securityOrigin
    * */
-  crossoriginWarning: 'A preconnect `<link>` was found for "{securityOrigin}" but was not used ' +
+  crossoriginWarning: 'A `<link rel=preconnect>` was found for "{securityOrigin}" but was not used ' +
     'by the browser. Check that you are using the `crossorigin` attribute properly.',
   /** A warning message that is shown when found more than 2 preconnected links */
-  tooManyPreconnectLinksWarning: 'More than 2 preconnect links were found. ' +
-   'Preconnect links should be used sparingly and only to the most important origins.',
+  tooManyPreconnectLinksWarning: 'More than 2 `<link rel=preconnect>` were found. ' +
```
Comment (patrickhulce) [inline lighthouse-core/audits/uses-rel-preconnect.js:48]:
```suggestion
  tooManyPreconnectLinksWarning: 'More than 2 `<link rel=preconnect>` directives were found. ' +
```

here's where that "directive" is useful I think :)

**Review by patrickhulce — APPROVED** [review]

File: lighthouse-core/audits/autocomplete.js [hunk 2]
```diff
@@ -18,24 +18,24 @@ const log = require('lighthouse-logger');
 
 const UIStrings = {
   /** Title of a Lighthouse audit that lets the user know if there are any missing or invalid autocomplete attributes on page inputs. This descriptive title is shown to users when all input attributes have a valid autocomplete attribute. */
-  title: 'Input elements use autocomplete',
+  title: '`<input>` elements use `autocomplete`',
```
Comment (patrickhulce) [inline lighthouse-core/audits/autocomplete.js:21]:
```suggestion
  title: '`<input>` elements correctly use `autocomplete`',
```

based on the failure title and description this feels more accurate?

File: lighthouse-core/audits/uses-rel-preconnect.js (hunk 1 above)
Comment (connorjclark) [reply lighthouse-core/audits/uses-rel-preconnect.js:48]: Is this necessary? Do you think "directives", as a computing term, can be translated to many languages?

Comment (connorjclark) [reply lighthouse-core/audits/uses-rel-preconnect.js:48]: ah, let's find out.

Comment (patrickhulce) [reply lighthouse-core/audits/uses-rel-preconnect.js]: 😆 I thought you wanted to use directive!

It seems like we need *some* noun here for the plural of this, but if you disagree I'm fine to roll with it

File: lighthouse-core/audits/uses-rel-preconnect.js [hunk 3]
```diff
@@ -36,17 +36,17 @@ const UIStrings = {
...
    * @description A warning message that is shown when the user tried to follow the advice of the audit, but it's not working as expected. Forgetting to set the `crossorigin` HTML attribute, or setting it to an incorrect value, on the link is a common mistake when adding preconnect links.
    * @example {https://example.com} securityOrigin
    * */
-  crossoriginWarning: 'A preconnect `<link>` was found for "{securityOrigin}" but was not used ' +
+  crossoriginWarning: 'A `<link rel=preconnect>` was found for "{securityOrigin}" but was not used ' +
     'by the browser. Check that you are using the `crossorigin` attribute properly.',
   /** A warning message that is shown when found more than 2 preconnected links */
-  tooManyPreconnectLinksWarning: 'More than 2 preconnect links were found. ' +
-   'Preconnect links should be used sparingly and only to the most important origins.',
+  tooManyPreconnectLinksWarning: 'More than 2 `<link rel=preconnect>` directives were found. ' +
```
Comment (patrickhulce) [inline lighthouse-core/audits/uses-rel-preconnect.js:48]: oooh what about "connections"! the point is you shouldn't be opening the connections and that's a much more obvious noun to translate?
```suggestion
  tooManyPreconnectLinksWarning: 'More than 2 `<link rel=preconnect>` connections were found. ' +
```

**Pull request merged**